frontend/src/views/CommentView.svelte

```diff
@@ -111,9 +114,23 @@
 <div class="container">
   <div class="inputs">
     <p>Add a comment to this resource.</p>
+    <details>
+      <summary>More Info</summary>
+      <p>Create links to other resources with the syntax:</p>
+      <pre style="color:var(--comment-color)">    #resourceId@hexAddress</pre>
+      <pre>    <span style="color:var(--comment-color)">#resourceId</span
+        >: The id of the resource to link to found in the url for that resource.</pre>
+      <pre>    <span style="color:var(--comment-color)">@hexAddress</span
+        >: (Optional) An address in the given resource.</pre>
```

Comment:
doing one pre tag forces us to use lint ignore tags because it has to be indented to the left margin. I think it looks cleaner this way, but can do one tag if its wanted.